add mgmt cluster docs for atip (#65)
suse-edge/suse-edge.github.io: docs/product/atip/requirements.md

##   Services (DHCP, DNS, etc)
External services required by ATIP, common options for DHCP, DNS, etc.

> ## Disable rebootmgr strategic 

Reviewer: ```suggestion
## Disable rebootmgr
```

> For telco workloads is really important to disable the rebootmgr strategic in order to avoid the reboot of the nodes in case of updates scheduled by the system.

Reviewer: ```suggestion
For telco workloads is really important to disable rebootmgr in order to avoid the reboot of the nodes in case of updates scheduled by the system.
```

> You could verify the strategic status using the next command:

Reviewer: ```suggestion
You could verify the strategy being used as:
```

```bash
cat /etc/rebootmgr.conf
[rebootmgr]
window-start=03:30
window-duration=1h30m
strategy=best-effort
lock-group=default
```

> and you could disable it using the next command:

Reviewer: ```suggestion
and you could disable it as:
```

```bash
sed -i 's/strategy=best-effort/strategy=off/g' /etc/rebootmgr.conf
```
or using the rebootmgrctl command:

```bash
rebootmgrctl strategy off
```

> > For more information about rebootmgr, please check: https://github.com/SUSE/rebootmgr

Reviewer: It would be better if you put this first and add a brief explanation... otherwise users like myself that doesn't know what is will be wondering why I need to disable this if I don't even know what it does.
